packages/accounts-controller/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Bump dependency on `@metamask/keyring-controller` to ^8.0.0
```

Comment:
We should release this as well, as a major version. Otherwise we'd have no way to use it without triggering peerDependency warnings

Reply:
done